Documentation/tutorial/toc.yml

```
@@ -19,9 +19,9 @@
     href: intro_overwrite_files.md
   - name: Links and Cross References
     href: links_and_cross_references.md
-- name: Template
+- name: DocFx Template System
```

Since the whole document is all about DocFX, consider remove it from here and also line 5?
```suggestion
- name: Template System
```

---

Yes, agree, I like that. I'll do a follow up commit and change those as well..